include/Util/CxtStmt.h

```diff
@@ -108,7 +108,7 @@ class CxtStmt
     /// Dump CxtStmt
     inline void dump() const
     {
-        SVFUtil::outs() << "[ Current Stmt: " << SVFUtil::getSourceLoc(inst) << " " << *inst << "\t Contexts: " << cxtToStr() << "  ]\n";
+        // TODO-os SVFUtil::outs() << "[ Current Stmt: " << SVFUtil::getSourceLoc(inst) << " " << *inst << "\t Contexts: " << cxtToStr() << "  ]\n";
```

Comment:
I think we can use value2String(inst) here for printing. All other TODO-os are the same?

Reply:
Fixed.